sql/hive/src/test/scala/org/apache/spark/sql/hive/MetastoreDataSourcesSuite.scala

```diff
@@ -18,11 +18,14 @@
 package org.apache.spark.sql.hive
 
 import java.io.File
+import java.sql.DriverManager
```

Comment:
Currently SaveAsTable is supported in Hive context only, Also SchemaRelationProvider interface is impletemented along with CreatableRelationProvider.